examples/manual/longer.md

```diff
@@ -0,0 +1,54 @@
+# End to end TMT tutorial
+## Setup
+1. Install tmt
+```bash
+sudo dnf install -y tmt-all
+```
+2. Create and switch to a brand new git repo
+```bash
+cd `mktemp -d`
+git init
+```
+## Test steps
+#### Step
```

Comment:
why this is level 4?

Reply:
Will move to level 3, but I prefer to leave one level for user to use as/if they want, especially in larger documents it helps navigation